dart/lib/sentry.dart

```diff
@@ -8,5 +8,6 @@ export 'src/scope.dart';
 export 'src/sentry.dart';
 export 'src/sentry_client.dart';
 export 'src/sentry_options.dart';
+export 'src/sentry_stack_trace_factory.dart';
```

Comment:
we should not expose this, this is the implementation detail

Reply:
✅